### PR TITLE
runtime: add lang and tracer_version tags to runtime metrics

### PR DESCRIPTION
**Pull request by jd**

### CODE REVIEW

File: ddtrace/internal/runtime/constants.py [hunk 1]
```diff
@@ -32,6 +32,8 @@
 ENV = 'env'
 LANG_INTERPRETER = 'lang_interpreter'
 LANG_VERSION = 'lang_version'
+LANG = 'lang'
```
Comment (majorgreys) [inline ddtrace/internal/runtime/constants.py]: Being that the metric names have `python` in them, what is a good motivation for adding a `lang` tag?

Comment (jd) [reply ddtrace/internal/runtime/constants.py]: Not sure. Would it help search or something like that?

Comment (brettlangdon) [reply ddtrace/internal/runtime/constants.py]: `lang` with `tracer_version` since `dd-trace-js` and `dd-trace-py` both have an `0.15.0`

Comment (brettlangdon) [reply ddtrace/internal/runtime/constants.py]: oh, sorry, read the comment wrong.... it depends, if we ever submit metrics without `python` in the metric name

File: ddtrace/internal/runtime/tag_collectors.py [hunk 2]
```diff
@@ -35,13 +37,20 @@ class PlatformTagCollector(RuntimeTagCollector):
...
+      - e.g. '0.29.0'
     """
-    required_modules = ['platform']
+    required_modules = ('platform', 'ddtrace')
 
     def collect_fn(self, keys):
         platform = self.modules.get('platform')
+        ddtrace = self.modules.get('ddtrace')
         tags = [
+            (LANG, 'Python'),
```
Comment (brettlangdon) [inline ddtrace/internal/runtime/tag_collectors.py]:
```suggestion
            (LANG, 'python'),
```

it is going to get lowercased anyways

File: ddtrace/internal/runtime/constants.py (hunk 1 above)
Comment (majorgreys) [reply ddtrace/internal/runtime/constants.py]: All runtime metrics a prefixed with `runtime.python` for now. Maybe this changes in the future. My take is that `lang` would be redundant here and so we should remove it.

**Comment by jd** [comment]
Lowercased the Python tag. I mean the python tag.

File: ddtrace/internal/runtime/constants.py (hunk 1 above)
Comment (brettlangdon) [reply ddtrace/internal/runtime/constants.py]: #1068 :)

Comment (brettlangdon) [reply ddtrace/internal/runtime/constants.py]: uses the same dogstatsd client, which sets these constant tags.

**Pull request merged**